### ISSUE TITLE
I made a zh-hans locals csv

### ISSUE THREAD

**Issue by Seicing** [Woprok/AOE4-AdvancedGameSettings#19]
If possible, I would like to be able to integrate into the mod
[Advanced Game Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8552734/Advanced.Game.Settings_zh-hans.csv)

**Comment by Woprok** [comment]
@Seicing Hi, its definitely possible. You will need to update it once I finish next update.

**Comment by Seicing** [comment]
> @Seicing Hi, its definitely possible. You will need to update it once I finish next update.

yes，I will keep it update

**Comment by Seicing** [comment]
> @Seicing Hi, its definitely possible. You will need to update it once I finish next update.
[Advanced Game Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8569106/Advanced.Game.Settings_zh-hans.csv)

Update for current version

**Comment by Woprok** [comment]
@Seicing I finished update for the mod, if you could update localization, I would appreciate it.

**Comment by Seicing** [comment]
> @Seicing I finished update for the mod, if you could update localization, I would appreciate it.

i will made it immediately

**Comment by Seicing** [comment]
> @Seicing I finished update for the mod, if you could update localization, I would appreciate it.
[Advanced Game Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8647299/Advanced.Game.Settings_zh-hans.csv)
Done

**Comment by Seicing** [comment]
> @Seicing I finished update for the mod, if you could update localization, I would appreciate it.
[Advanced Game Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8647323/Advanced.Game.Settings_zh-hans.csv)
done，Maybe need a little correction after the actual test

**Comment by Seicing** [comment]
> @Seicing I finished update for the mod, if you could update localization, I would appreciate it.
[Advanced Game Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8647451/Advanced.Game.Settings_zh-hans.csv)
finished and tested

**Comment by Woprok** [comment]
@Seicing I think I was finally able to upload the mod... Check if it works ingame.

**Comment by Seicing** [comment]
> @Seicing我想我终于可以上传模组了……检查它是否可以在游戏中运行。

I try to update but it crashed，Localization seems have a problem

**Comment by Seicing** [comment]
> @Seicing I think I was finally able to upload the mod... Check if it works ingame.

working correctly now, excellent work!

**Comment by chifeiyen** [comment]
[Advanced.Game.Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8651623/Advanced.Game.Settings_zh-hans.csv)

Hello @Woprok @Seicing,
I have fixed and adjusted parts of translation and format of the Simplified Chinese text.
Attached is my updated file, for your check. Thanks!

**Comment by Seicing** [comment]
> [Advanced.Game.Settings_zh-hans.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/8651623/Advanced.Game.Settings_zh-hans.csv)
> 
> 你好@Woprok @Seicing, 今天和 Seicing 聊了聊，帮助修复和调整简体中文文本的翻译和格式。 附件是我的更新文件，供您检查。谢谢！
> 
> 变更日志：
> 
> 1. 修复并简化某些部分的翻译。
> 2. 调整格式、排版和标点符号，使其与官方中文文本保持一致。

good

**Comment by Seicing** [comment]
same problem as before crashed.

**Comment by Woprok** [comment]
@Seicing crashing ? I am ware it was not possible to download it, but that should work now

**Comment by Seicing** [comment]
> @Seicing crashing ? I am ware it was not possible to download it, but that should work now

no crash in gaming，just can't download it，my description is wrong

**Comment by Woprok** [comment]
At least its fixed until I try to push next update and mod breaks again 😢

**Comment by Seicing** [comment]
> At least its fixed until I try to push next update and mod breaks again 😢

it works correctly now，Thanks for the quick fix

**Comment by Woprok** [comment]
@Seicing @w3k5 @Helly0000 
Profanity filter is now creating issues with translations and prevents mod upload. 
Both Russian and Chinese localization are disabled.

I would appreciate if you could update Chinese & Russian localizations and test if game allows them to be published. Once you are sure it allows it, create pull request or throw new localization file here. I will then update main mod.

**Comment by Helly0000** [comment]
> @Seicing @w3k5 @Helly0000 Profanity filter is now creating issues with translations and prevents mod upload. Both Russian and Chinese localization are disabled.
> 
> I would appreciate if you could update Chinese & Russian localizations and test if game allows them to be published. Once you are sure it allows it, create pull request or throw new localization file here. I will then update main mod.

My English is poor and I didn't get it. You mean the profanity filter works improperly on Chinese language therefore the MOD can't be uploaded?
I do encounter something wrong while uploading my mod which is based on your AGS. I can't finished uploading then I killed the game exe. And then I found the mod was published anyway however the name was changed. I thought it was AOE4 server. But now I believe it is the profanity filter.

**Comment by Helly0000** [comment]
![image](https://user-images.githubusercontent.com/35934370/169678058-e75da2dd-c513-4580-86a3-178533bc9a70.png)
That is weird. My mod's name is 防守图测试（基于高级游戏设置）(A PVE mod testing based on AGS). But the uploaded mod's name is 这把我们四个真是太厉害了！(We four are so great this time!). I even don't know who wrote this words...

**Comment by Woprok** [comment]
> My English is poor and I didn't get it. You mean the profanity filter works improperly on Chinese language therefore the MOD can't be uploaded? I do encounter something wrong while uploading my mod which is based on your AGS. I can't finished uploading then I killed the game exe. And then I found the mod was published anyway however the name was changed. I thought it was AOE4 server. But now I believe it is the profanity filter.

Yeah, profanity filter is checking every mod that is being uploaded. Then it creates report and prevents mod from being updated, if it finds anything. (Relic devs fixed issue where upload process was previously crashing the game, so now we have to deal with profanity filter blocking uploads.)

**Comment by Zyiha9755** [comment]
Hello, what should I do if I want to add a traditional Chinese interface? Thank you.

**Comment by TonyPL90** [comment]
@Woprok Hello, I don't know if you still support this mod, but I decided to translate it into Polish. If you could integrate it with the mod, that would be great, it would make the settings easier for me and my friends.
[Advanced Game Settings_pl.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/15360422/Advanced.Game.Settings_pl.csv)

**Comment by Woprok** [comment]
> @Woprok Hello, I don't know if you still support this mod, but I decided to translate it into Polish. If you could integrate it with the mod, that would be great, it would make the settings easier for me and my friends. [Advanced Game Settings_pl.csv](https://github.com/Woprok/AOE4-AdvancedGameSettings/files/15360422/Advanced.Game.Settings_pl.csv)

Hi, I am no longer maintaining the mod, you can create your own fork or ask someone who maintains fork of the mod to include it in his
